docs/user-guide/src/capm3/automated_cleaning.md

```diff
@@ -0,0 +1,35 @@
+# Automated Cleaning
+
+Before reading this page, please see [Baremetal Operator Automated Cleaning](../bmo/automated_cleaning.md) page.
+
+If you are using only Metal3 Baremetal Operator, you can skip this page and refer to Baremetal
+Operator automated cleaning [page](../bmo/automated_cleaning.md) instead.
+
+For deployments following Cluster-api-provider-metal3 (CAPM3) workflow, automated cleaning can
+be (recommended) configured via CAPM3 custom resources (CR).
+
+There are two automated cleaning modes available  which can be set via `automatedCleaningMode` field of a
+Metal3MachineTemplate `spec` or Metal3Machine `spec`.
+
+- `metadata` to enable the cleaning
+- `disabled` to disable the cleaning
+
+When enabled (`metadata`), automated cleaning kicks off when a node is in the first provisioning and on every deprovisioning.
+There is no defaulting for `automatedCleaningMode` in Metal3MachineTemplate and Metal3Machine. If user doesn't set any mode,
+the field in the `spec` will be omitted. Unsetting `automatedCleaningMode` in the Metal3MachineTemplate will block the synchronization of the cleaning mode between the Metal3MachineTemplate and Metal3Machines.
```

Comment:
Maybe add here that "this enables the selective operations described below" or "this allows the Metal3Machines to have different values if desired"?
It wasn't clear to me before reading about the selective operations if "blocking the synchronization" was a "configuration error" or something that may be desired in some situations. With this addition I think it would be clearer.

Reply:
good point. Added